database/tableau_api_models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+from database.common_models import AbstractModel
+from .study_models import Study
+from .user_models import Participant
+
+
+class SchemaTableauAPI(AbstractModel):
```

Comment:
Can we rename this to something that conveys better what the data represents? Like `SummaryStatisticDaily` maybe? I expect this may change again later, but I think it's worth making the first change.